train-winrm.gemspec

```diff
@@ -4,7 +4,7 @@
 
 # It is traditional in a gemspec to dynamically load the current version
 # from a file in the source tree.  The next three lines make that happen.
-lib = File.expand_path("../lib", __FILE__)
+lib = File.expand_path("../lib", __FILE__) # rubocop:disable Style/ExpandPathArguments
```

Comment:
👍  to this exception, something iffy is happening when trying to correct this violation. I just opened https://github.com/inspec/train-winrm/issues/26 to get us onto chefstyle - I'll leave a comment that we should address this exception at the same time.